lib/alchemy/tasks/helpers.rb

```diff
@@ -22,18 +22,18 @@ def database_config
 
       private
 
-      def database_command(adapter, action='import')
+      def database_command(adapter, action = 'import')
         case adapter.to_s
-        when /mysql/
-          "#{mysql_command(mysql_command_for(action))} #{database_config['database']}"
-        when /postgresql/
-          "#{postgres_command(postgres_command_for(action))} #{database_config['database']}"
+          when /mysql/
+            "#{mysql_command(mysql_command_for(action))} #{database_config['database']}"
+          when /postgresql/
+            "#{postgres_command(postgres_command_for(action))} #{database_config['database']}"
         else
-          raise ArgumentError, "Alchemy only supports #{action}ing MySQL and PostgreSQL databases. #{adapter} is not supported."
+            raise ArgumentError, "Alchemy only supports #{action}ing MySQL and PostgreSQL databases. #{adapter} is not supported."
```

Comment:
Metrics/LineLength: Line is too long. [130/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)